source/release-notes.txt

```diff
@@ -10,6 +10,39 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.37.0
+----------------
+
+*Released May 25, 2023*
+
+New Features:
+
```

Comment:
(question)
Is there a ticket for the dark mode update?

Reply:
I could not find one.

Reply:
Ok! Thanks for looking!